apis/bulkupload/bulkserviceapi.yaml

```diff
@@ -75,6 +75,61 @@ paths:
           description: >-
             Access token of registered user performing given API request.
 
+      responses:
+        '200':
+          description: 'OK ! Successful operation."bulk upload user" operation was successfully executed.'
+          schema:
+            $ref: '#/definitions/BulkUploadResponse'
+        '400':
+          description: 'BAD REQUEST. The "bulk upload user" operation failed ! The possible reason for failure is that you may have missed providing input for a mandatory parameter.'
+          schema:
+            $ref: '#/definitions/BulkUploadErrorResponse'
+        '500':
+          description: 'INTERNAL SERVER ERROR ! Looks like something went wrong! These errors are tracked automatically, but if the problem persists feel free to contact us. In the meantime, try refreshing.'
+  /user/v2/bulk/upload:
+    post:
+      description: >-
+        This API is associated with upload shadow users in bulk on the Sunbird Platform,shadow user are those who want to migrate from custodian org to non-custodian org.
```

Comment:
This API is associated with uploading shadow user on the Sunbird Platform.Only org admin role can upload user. If any uploaded user identifier (email,phone) matched with custodian user then he/she will be eligible for migration , from custodian to state user.

Reply:
done